frontend/src/widgets/HorizontalTableWidget.scss

```diff
@@ -74,4 +78,22 @@
     text-decoration: none;
     font-weight: bold;
     color: $base-darkest;
-  }
\ No newline at end of file
+  }
+
+  .smarthub-horizontal-table-widget .usa-checkbox__label {
+    padding-left: 0;
+  }
+
+  a {
```

Comment:
we need to add some qualification to this selector or it targets every "a" element on the site

Reply:
Also, we can abstract

```css
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

to a helper class

Reply:
This has been updated.